mappings/net/minecraft/entity/projectile/FishingBobberEntity.mapping

```diff
@@ -23,4 +30,12 @@ CLASS net/minecraft/class_1536 net/minecraft/entity/projectile/FishingBobberEnti
 		ARG 3 lureLevel
 		ARG 4 luckOfTheSeaLevel
 	METHOD method_6947 getOwner ()Lnet/minecraft/class_1657;
+	METHOD method_6949 tickFishingLogic (Lnet/minecraft/class_2338;)V
```

Comment:
I believe we tend to use `update` rather than `tick`, so that would be `updateFishingLogic`.

Reply:
we prefer tick over update.

Reply:
https://github.com/FabricMC/yarn/blob/1.14.4/CONVENTIONS.md#ticks-and-updates
Looks like it. Although a lot of stuff in entities doesn't obey this rule.

Reply:
I went for `tick` because it's called from the `tick` method whenever the bobber is in water (so effectively a conditional tick method); does that change your assessment?

Reply:
It seems that `tick` is indeed the better choice according to conventions.